CONTRIBUTING.md

```diff
@@ -53,9 +53,33 @@ Please make sure you have:
 3. All branches have the following naming format: `iox-#[issue]-branch-name` e.g. `iox-#123-super-duper-feature`
 4. All commits have the following naming format: `iox-#[issue] commit message` e.g. `iox-#123 implemented super-duper feature`
 5. All commits have been signed with `git commit -s`
-6. You open your pull request towards the base branch `staging`
+6. You open your pull request towards the base branch `master`
 7. Link the pull request to the according Github issue and set the label accordingly
 
+## Branching strategy
+
+`master`
+
+* Main development branch
+* Open for external contributions
+
+`release_x.y`
+
+* Branch for stablising a certain release
+* Write access limited to maintainers
+* Fine-tuning of external contribution e.g. running Axivion SCA
+* Finish any missing implementations regarding the quality levels
+
+As depicted below, after the release branch has been created the stabilisation phase will begin. After finishing the release, a git tag will be created to point to `HEAD` of the release branch. Follow-up releases will be branched off from the git tag.
+
+```
+o---o---o---o---o  master
+     \
+      o---o---o---o---o  release_1.0 / v1.0 tag
+                       \
+                        o---o---o  release_1.1
+```
```

Comment:
Just for clarification as it looks like the branch is abandoned after the tag. I know it usually this way
```
o---o---o---o---o  master
     \
      \      v1.0.0      v1.0.1
       \        |           |
        o---o---o---o---o---o---o  release_1.0
                         \
                          \      v1.1.0
                           \        |
                            o---o---o  release_1.1
```
Do you mean the same?

Well, after some thinking I usually know it like this
```
o---o---o---o---o---o---o---o---o---o---o  master
     \                           \
      \      v1.0.0      v1.0.1   o---o---o release_1.1
       \        |           |
        o---o---o---o---o---o---o  release_1.0
```
But since we might easily introduce ABI changes in master, I should be more practical to do it like you proposed

Reply:
Yep, I meant the first approach. Adapted the docu to make it more clear.